### DISCUSSION TITLE
gmails layout doesn't look as preview (removing `padding-right: 24px;` fixes it)

### DISCUSSION THREAD

**Discussion by HaruMiyaGi** [bootstrap-email/bootstrap-email#164] [Help]
Hello, I have this simple layout I'd like to use:

```html
<body class="bg-light">
	<div class="container p-8">
		<div class="p-5 bg-secondary">
			<div class="row">
				<div class="col-12 text-center">
					<div class="card max-w-96">
						<div class="card-body">
							<hr class="mt-2 mb-5" />
							<div class="row">
								<div class="col-6 text-left">Left</div>
								<div class="col-6 text-right">Right</div>
							</div>
							<hr class="mt-2 mb-5" />
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</body>
```

When I build this template it looks fine when I view html file:
![preview](https://user-images.githubusercontent.com/66522128/154145191-c9a6a22e-1dd3-4cbd-b796-3400e5901746.png)

However, when I send this layout to gamil account it adds extra padding right, like so: 
![desktop-gmail](https://user-images.githubusercontent.com/66522128/154145295-d780ed17-2dcf-4fd1-be31-f8c162f64a6d.png)
![mobile-gmail](https://user-images.githubusercontent.com/66522128/154145304-bd1edb81-f978-49a6-b558-ed2c643d95ad.png)

Removing every instance of `padding-right: 24px;` from compiled template seems to fix it for gmail but it breaks it for other clients.

Am I doing my html wrong, or what can I do to fix this?

**Comment by stuyam** [comment]
Good question. The padding right is added because it needs to be there for gutter support, however some clients like gmail don't support negative margin which this relies on. So the tradeoff is some clients you get some extra right padding like this. By removing the `padding-right: 24px;` it will look correct for your use case but if you could see the outline of each column the right column will actually be wider than the other columns. You could do something like this:
```html
<div>
  <div class="ax-left">Left</div>
  <div class="ay-right>Right</div>
</div>
```
